apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -781,16 +781,16 @@ fill_defaults(Configs) when is_list(Configs) ->
 fill_defaults(Config) ->
     emqx_authn:check_config(Config, #{only_fill_defaults => true}).
 
-convert_certs(#{<<"ssl">> := SSLOpts} = Config) ->
+convert_certs(#{ssl := SSLOpts} = Config) ->
     NSSLOpts = lists:foldl(fun(K, Acc) ->
                                case maps:get(K, Acc, undefined) of
                                    undefined -> Acc;
                                    Filename ->
                                        {ok, Bin} = file:read_file(Filename),
                                        Acc#{K => Bin}
```

Comment:
If the intension is to *NOT* return file content in the HTTP response, shouldn't we just skip doing this?